src/main.rs

```diff
@@ -24,10 +25,12 @@ extern crate log;
 /// Fast Port Scanner built in Rust.
 /// WARNING Do not use this program against sensitive infrastructure since the
 /// specified server may not be able to handle this many socket connections at once.
+/// - Discord https://discord.gg/rAnvBbg
```

Comment:
This is the incorrect discord link- it has been deleted and invalidated.
Correct link should be https://discord.gg/GFrQsGy to align with the README.md
```suggestion
/// - Discord https://discord.gg/GFrQsGy
```

Reply:
fixed :)